### PR TITLE
[GPU] Add oneDNN zero point value debug log.

### PR DESCRIPTION
**Pull request by hyunback**
Signed-off-by: hyunback <hyunback.kim@intel.com>

### Tickets:
 - *8010*

### CODE REVIEW

File: src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp [hunk 1]
```diff
@@ -63,12 +65,38 @@ struct convolution_onednn : typed_primitive_onednn_impl<convolution, dnnl::convo
...
             args.insert({DNNL_ARG_ATTR_ZERO_POINTS | DNNL_ARG_SRC, a_zp->get_onednn_memory(desc)});
+
+            GPU_DEBUG_GET_INSTANCE(debug_config);
+            GPU_DEBUG_IF(debug_config->verbose >= 2) {
+                size_t desc_size = desc.get_size();
+                std::vector<int32_t> debug_buf(desc_size);
+                auto dnnl_mem = a_zp->get_onednn_memory(desc);
+                void *mapped_ptr = dnnl_mem.map_data();
+                if (mapped_ptr) {
+                    std::memcpy(debug_buf.data(), mapped_ptr, desc_size);
```
Comment (isanghao) [inline src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp:76]: is it necessary to memcpy? can we just access mapped_ptr?

File: src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp [hunk 2]
```diff
@@ -63,12 +65,38 @@ struct convolution_onednn : typed_primitive_onednn_impl<convolution, dnnl::convo
...
+                    GPU_DEBUG_COUT << instance.get_node().id() << " activations_zero_points: " << debug_buf[0] << std::endl;
```
Comment (isanghao) [inline src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp:77]: what about if it is not scalar but per-channel?

File: src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp (hunk 1 above)
Comment (hyunback) [reply src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp]: Use memcpy, just follow up the samples, It works well without memcpy, Update direct using mapped_ptr.

File: src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp (hunk 2 above)
Comment (hyunback) [reply src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp]: Updated to print all values from desc size.

**Review by isanghao — APPROVED** [review]

**Pull request merged**